invideoquiz/invideoquiz.py

```diff
@@ -48,7 +48,7 @@ class InVideoQuizXBlock(StudioEditableXBlockMixin, XBlock):
 
     timemap = String(
         display_name=_('Problem Timemap'),
-        default='',
+        default={},
```

Comment:
http://stackoverflow.com/questions/26320899/why-is-the-empty-dictionary-a-dangerous-default-value-in-python